presto-spi/src/main/java/com/facebook/presto/spi/Page.java

```diff
@@ -253,8 +253,8 @@ public Page mask(int[] retainedPositions)
     {
         requireNonNull(retainedPositions, "retainedPositions is null");
 
-        Block[] blocks = Arrays.stream(this.getBlocks())
-                .map(block -> new DictionaryBlock(block, retainedPositions))
+        Block[] blocks = Arrays.stream(getBlocks())
```

Comment:
In a separate commit, can you rename this method to `getPositions` to match the block method (see below)?